index.js

```diff
@@ -9,10 +9,10 @@ module.exports = ({
   qualifier = null, // specify a version or alias to invoke a published version of the function
   clientContext = null, // up to 3583 bytes of base64-encoded data about the invoking client to pass to the function in the context object
   lambdaProxy, // AWS lambda invocation proxy
-  ...lambdaProxyArgs,
+  ...lambdaProxyArgs
```

Comment:
I would recommend adding a `.prettierrc` file in the root to avoid PRs fucking up the formatting of your code.
For this specific line I would be in favour of the trailing comma.

> If you want to add a new property, you can simply add a new line without modifying the previously last line if that line already uses a trailing comma. This makes version-control diffs cleaner and editing code might be less troublesome.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

This is the `.prettierrc` setup I use in [faugra](https://github.com/zvictor/faugra) and recommend the most:

```js
{
  "printWidth": 100,
  "singleQuote": true,
  "trailingComma": "es5",
  "semi": false
}
```

Reply:
Hi @zvictor, I actually use https://standardjs.com for listing. I will add it to the CI/CD checks. 
Thanks